src/patternfly/components/Progress/docs/code.md

```diff
@@ -6,12 +6,12 @@ Note: If this component is describing the loading progress of a particular regio
 
 | Attribute | Applied To | Outcome |
 | -- | -- | -- |
-| `role="progressbar"` | `.pf-c-progress` |  This role is used for an element that displays the progress status for a task that takes a long time or consists of several steps. |
-| `aria-valuenow=""` | `.pf-c-progress` |  This value needs to be updated as progress continues. |
-| `aria-valuemin="0"` | `.pf-c-progress` |  The minimum value for the progress bar. |
-| `aria-valuemax="100"` | `.pf-c-progress` |  The maximum value for the progress bar. If the progress is only defined using `aria-valuenow` (e.g a percentage), the value should be set to "100". If the progress is defined using `aria-valuetext`, then this value can be a number other than 100. For example, if `aria-valuetext` is "2 of 5 units", then `aria-valuemax` can be "5" and `aria-valuenow` can be "2". |
-| `aria-describedby="[id of .pf-c-progress__description]"` | `.pf-c-progress` |  The description of what progress is being shown. |
-| `aria-valuetext="[loading state]"` | `.pf-c-progress` |  Provide a text string that communicates current status. Only use if the important information about status is included in the text string. Do not use if percentage is the most important value to communicate. Some screen readers will ignore the percentage value determined from `aria-valuenow` when `aria-valuetext` is used. |
+| `role="progressbar"` | `.pf-c-progress__bar` |  This role is used for an element that displays the progress status for a task that takes a long time or consists of several steps. |
+| `aria-valuenow=""` | `.pf-c-progress__bar` |  This value needs to be updated as progress continues. |
+| `aria-valuemin="0"` | `.pf-c-progress__bar` |  The minimum value for the progress bar. |
+| `aria-valuemax="100"` | `.pf-c-progress__bar` |  The maximum value for the progress bar. If the progress is only defined using `aria-valuenow` (e.g a percentage), the value should be set to "100". If the progress is defined using `aria-valuetext`, then this value can be a number other than 100. For example, if `aria-valuetext` is "2 of 5 units", then `aria-valuemax` can be "5" and `aria-valuenow` can be "2". |
+| `aria-describedby="[id of .pf-c-progress__description]"` | `.pf-c-progress__bar` |  The description of what progress is being shown. |
+| `aria-valuetext="[loading state]"` | `.pf-c-progress__bar` |  Provide a text string that communicates current status. Only use if the important information about status is included in the text string. Do not use if percentage is the most important value to communicate. Some screen readers will ignore the percentage value determined from `aria-valuenow` when `aria-valuetext` is used. |
```

Comment:
Can you add a row for including `aria-hidden="true"` on the `.pf-c-progress__status` element?

The explanation can be something like "Hides the visible progress bar status from screen readers. This information is communicated by the aria attributes defined on the `pf-c-progress__bar` element." Feel free to revise that.